.drone.yml

```diff
@@ -77,6 +77,9 @@ pipeline:
   before-release:
     image: fpfis/httpd-php-dev:7.1
     commands:
+      # Rebuild styles without Source map for releases.
+      - npm rebuild node-sass
+      - NODE_ENV=production npm run build:styles
```

Comment:
What I meant was to do this directly on:
```
pipeline:
  npm-build:
    group: prepare
    image: node:9.5.0
    commands:
      - npm install --unsafe-perm
      - npm run build
```

So:
```
pipeline:
  npm-build:
    group: prepare
    image: node:9.5.0
    commands:
      - npm install --unsafe-perm
      - NODE_ENV=production npm run build
```

This is also theoretically more correct since we will test a prod build.